terraform/Makefile

```diff
@@ -2,8 +2,12 @@ SHA = $(shell git rev-parse --short HEAD)
 RUN = docker-compose run --rm
 
 ##TASKS
+
+s3_init:
+	docker-compose run --rm aws \
+	s3api create-bucket --bucket wordpress-s3 --create-bucket-configuration LocationConstraint=ap-southeast-2 --acl private
```

Comment:
Could the parameters be variables?

Reply:
Hey @Faria-Marcio, How did you mean?
Like the region / bucket name ?